src/luavariant.h

```diff
@@ -0,0 +1,19 @@
+#ifndef FS_LUAVARIANT_H
+#define FS_LUAVARIANT_H
+
+#include <variant>
+
+using LuaVariant = std::variant<uint32_t, Position, Position, std::string>;
```

Comment:
Having the `Position` twice breaks `std::get<type>`, that's why I'm using `std::get<index>` everywhere